app/assets/stylesheets/copilot.sass

```diff
@@ -0,0 +1,15 @@
+#copilot_wide
```

Comment:
since height and width are the same for copilot and copilot_wide, copilot_wide could extend copilot and add the background property